src/wasm/wasm-validator.cpp

```diff
@@ -75,6 +76,7 @@ struct ValidationInfo {
 
   template <typename T, typename S>
   std::ostream& fail(S text, T curr, Function* func) {
+    __builtin_trap();
```

Comment:
what is this?

Reply:
Debug break that I forgot to remove. Removed in 
6f85f55